device_doctor/lib/src/ios_device.dart

```diff
@@ -47,6 +48,15 @@ class IosDeviceDiscovery implements DeviceDiscovery {
     return results;
   }
 
+  /// iOS device property check is not needed at this moment.
+  ///
+  /// But we will implement this to replace existing chromium side bot config logic
+  /// after devicelab migration to LUCI is done.
```

Comment:
Update the docs to describe what the method does instead? given this is a method on a class it can be used for different use cases that the one described.

Reply:
Updated.